website/docs/how-to/unleash-edge-quickstart.md

```diff
@@ -0,0 +1,536 @@
+
+
+## How to run Unleash Edge locally
+
+First, make sure your Unleash instance is running (locally or remotely) and generate a new backend API token.
+
+
+<Tabs groupId="method">
+
+  <TabItem value="github-release" label="GitHub Release installer" default>
+    You can install prebuilt binaries for your OS directly from GitHub Releases:
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -LsSf https://github.com/Unleash/unleash-edge/releases/download/<version>/unleash-edge-installer.sh | sh
+    ```
+
+    :::info[GitHub Releases versioning]
+
+    Make sure to replace `<version>` with a valid GitHub Release tag such as `v20.1.0`.
+
+    You can pick a valid version from the list of [releases on GitHub](https://github.com/Unleash/unleash-edge/releases).
+
+    :::
+
+    Then launch it:
+
+    ```shell
+    unleash-edge edge \
+      --strict \
+      --upstream-url <your_unleash_instance> \
+      --tokens '<your_backend_token>'
+    ```
+
+  </TabItem>
+
+  <TabItem value="rust" label="Rust toolchain">
+    If you're comfortable with the Rust toolchain, install the CLI with [cargo binstall](https://github.com/cargo-bins/cargo-binstall?tab=readme-ov-file#installation) (or [from source](https://github.com/Unleash/unleash-edge/blob/main/docs/development-guide.md)):
+
+    ```shell
+    cargo binstall unleash-edge
+    ```
+
+    Then launch it:
+
+    ```shell
+    unleash-edge edge \
+      --strict \
+      --upstream-url <your_unleash_instance> \
+      --tokens '<your_backend_token>'
+    ```
+
+  </TabItem>
+
+  <TabItem value="docker" label="Docker">
+    Launch Unleash Edge locally with Docker:
+
+    ```shell
+    docker run -it \
+      -p 3063:3063 \
+      -e STRICT=true \
+      -e UPSTREAM_URL=<your_unleash_instance> \
+      -e TOKENS='<your_backend_token>' \
+      unleashorg/unleash-edge \
+      edge
+    ```
+
+    :::info[Docker Image versioning]
+
+    By default, the command above uses the latest `unleashorg/unleash-edge` tag.
+
+    You can also pick a specific version from the list of [tags on Docker Hub](https://hub.docker.com/r/unleashorg/unleash-edge).
+
+    :::
+
+  </TabItem>
+
+  <TabItem value="docker-compose" label="Docker Compose">
+    Launch the [examples/docker-compose.yml](https://github.com/Unleash/unleash-edge/blob/main/examples/docker-compose.yml) file:
+
+    ```shell
+    git clone https://github.com/unleash/unleash-edge/
+    cd unleash-edge/examples
+    docker compose up
+    ```
+
+    :::info[Docker Compose services]
+
+    Please note that the sample Docker Compose file includes a Redis service for caching.
+    In case you want to run Unleash Edge without Redis, make sure to remove the `REDIS_URL` environment variable and the Redis service.
+
+    :::
+
+  </TabItem>
+</Tabs>
+
+#### Required parameters
+
+Let's break down the parameters you need to replace in the launch command.
+
+### `<your_unleash_instance>`
```

Comment:
I'd vote against code formatting in headings, unless necessary. If you want to keep them, then we need to fix the hierarchy, so these should be l4

https://developers.google.com/style/headings#heading-and-title-format

Reply:
Updated 👌